frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/when-to-cut/when-to-cut.component.html

```diff
@@ -36,7 +36,7 @@ <h3>When to cut firewood</h3>
   <p>Firewood permits are valid through the calendar year.</p>
   <p>See <a href="#prohibited">prohibited areas and restrictions</a> for more information.</p>
   <h3>Cost</h3>
-  <p>Firewood permits cost <strong>$20.00 for 6 cords of wood</strong>. All permits are nonrefundable</p>
+  <p>Firewood permits cost <strong>$20.00 for 6 cords of wood</strong>. Limit of (waiting on the number from POC) cords per household per year. All permits are nonrefundable</p>
```

Comment:
Similar question re: parenthetical. If we don't have this information yet, is there an issue to come back and fill it in?

Reply:
Removed this one too.